src/components/PostPageDetail/index.js

```diff
@@ -660,33 +672,36 @@ const PostPageDetailIdComponent = (props) => {
                   isLoading={loadingPost}
                   refreshComment={handleRefreshComment}
                   refreshChildComment={handleRefreshChildComment}
-                  navigateToReplyView={(data) =>
-                    navigateToReplyView(
-                      data,
-                      updateParentPost,
-                      findCommentAndUpdate,
-                      item,
-                      updateVoteLatestChildren,
-                      getComment
-                    )
-                  }
                   findCommentAndUpdate={findCommentAndUpdate}
                   contextSource={contextSource}
                   updateVote={handleUpdateVote}
                   isShortText={isShortText}
+                  onReplyButtonClick={(reactiondId, username, level) => {
+                    if (level === 0 || level === 1) {
+                      setTypeComment('child');
+                    }
+                    setReactionId(reactiondId);
+                    setReplyUsername(username);
+                  }}
                 />
               )
             )}
           </ScrollView>
 
           <WriteComment
             postId={feedId}
-            username={handleUserName(item)}
+            username={replyUsername || handleUserName(item)}
             value={textComment}
             onChangeText={(value) => setTextComment(value)}
             onPress={onComment}
             loadingPost={loadingPost}
             isKeyboardOpen={isKeyboardOpen}
+            onClear={() => {
+              setTextComment('');
+              setTypeComment('parent');
+              setReplyUsername('');
+            }}
+            isReply={typeComment === 'child' && replyUsername}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `WriteComment` component now accepts new props (`isReply`, `onClear`). These changes seem to be part of a larger refactor of the commenting functionality. Ensure that the `WriteComment` component can handle these new props.
```diff
+ username={replyUsername || handleUserName(item)}
+ onClear={() => {
+   setTextComment('');
+   setTypeComment('parent');
+   setReplyUsername('');
+ }}
+ isReply={typeComment === 'child' && replyUsername}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->